admin/orders.php

```diff
@@ -968,13 +962,13 @@ function couponpopupWindow(url) {
 
                     $show_payment_type = $orders->fields['payment_module_code'] . '<br>' . $orders->fields['shipping_module_code'];
                     ?>
-                <td class="dataTableContent" align="right"><?php echo $show_difference . $orders->fields['orders_id']; ?></td>
-                <td class="dataTableContent" align="left" width="50"><?php echo $show_payment_type; ?></td>
+                <td class="dataTableContent right"><?php echo $show_difference . $orders->fields['orders_id']; ?></td>
```

Comment:
this should be
```
<td class="dataTableContent text-right">
```

Reply:
This is in the admin stylesheet
.right {
  text-align: right;
}

text-right is part of bootstrap. Is that preferred?

Reply:
That's what I used in the rest of the pages

Reply:
ok, I've modified them.